runtime/vara/src/lib.rs

```diff
@@ -615,7 +616,11 @@ parameter_types! {
     pub const BondingDuration: sp_staking::EraIndex = 14;
     // 41 eras during which slashes can be cancelled (slightly less than 7 days)
     pub const SlashDeferDuration: sp_staking::EraIndex = 13;
-    pub const MaxNominatorRewardedPerValidator: u32 = 256;
+    pub const MaxExposurePageSize: u32 = 512;
+    // Note: this is not really correct as Max Nominators is (MaxExposurePageSize * page_count) but
+    // this is an unbounded number. We just set it to a reasonably high value, 1 full page
+    // of nominators.
+    pub const MaxNominators: u32 = 512;
```

Comment:
wouldn't this type of tx exceed our non-mandatory limits?
I think its better to keep 256 for both?